cmd/kubeadm/.import-restrictions

```diff
@@ -112,15 +112,7 @@
 				"github.com/coreos/etcd/pkg/tlsutil",
 				"github.com/coreos/etcd/pkg/transport",
 				"github.com/davecgh/go-spew/spew",
-				"github.com/docker/distribution/digestset",
 				"github.com/docker/distribution/reference",
-				"github.com/docker/docker/api",
-				"github.com/docker/docker/client",
-				"github.com/docker/docker/pkg/term",
-				"github.com/docker/go-connections/nat",
-				"github.com/docker/go-connections/sockets",
-				"github.com/docker/go-connections/tlsconfig",
-				"github.com/docker/go-units",
 				"github.com/docker/libnetwork/ipvs",
```

Comment:
[1]

Reply:
This one is mostly pulling consts IIRC, I'll send a PR to consolidate our consts (rbac, this one and some others)

Reply:
This is mostly pulled to use consts IIRC, I'll send a PR to consolidate our const usage (rbac, kube-proxy etc..)